shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -1799,8 +1801,21 @@ private void sendWindowChangeEvent(@NonNull SemanticsNode route) {
       // next.
       routeName = " ";
     }
-    event.getText().add(routeName);
-    sendAccessibilityEvent(event);
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
+      setAccessibilityPaneTitle(routeName);
+    } else {
+      AccessibilityEvent event =
+          obtainAccessibilityEvent(route.id, AccessibilityEvent.TYPE_WINDOW_STATE_CHANGED);
+      event.getText().add(routeName);
+      sendAccessibilityEvent(event);
+    }
+  }
+
+  @TargetApi(28)
+  @RequiresApi(28)
+  @VisibleForTesting
```

Comment:
You can use: `ReflectionHelpers.setStaticField(Build.VERSION.class, "SDK_INT", <api-number>);`

http://robolectric.org/javadoc/3.0/org/robolectric/util/ReflectionHelpers.html

Reply:
I'm not sure why we haven't used this pattern yet, but maybe it's time to re-evaluate  `@VisibleForTesting` methods if they are working around this issue.

Reply:
I found a different way to mock the view, so i don't need the VisibleForTesting

Reply:
for some reason using Build.VERSION_CODES.P causes the Linux_unopt failure

Reply:
https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket.appspot.com/8840536145538768400/+/u/android_lint/dart_bin_main.dart/stdout

Reply:
that is weird. since the constant P = 28. My only guess is that we are using a jar prior to https://cs.android.com/android/_/android/platform/frameworks/base/+/3da5f3b2ddf35ed436cd9a0c6a349c59c97f8d78

cc @dnfield is it controlled here https://github.com/flutter/engine/blob/master/tools/android_lint/bin/main.dart#L77 ?